go/tosca/types_test.go

```diff
@@ -87,15 +87,61 @@ func TestAddress_JSON_InvalidValueDecodingFails(t *testing.T) {
 	}
 }
 
+func TestValue_NewValue(t *testing.T) {
+
+	tests := []struct {
+		value Value
+		index int
+	}{
+		{NewValue(1), 31},
+		{NewValue(1, 0), 23},
+		{NewValue(1, 0, 0), 15},
+		{NewValue(1, 0, 0, 0), 7},
+	}
+
+	for _, test := range tests {
+		t.Run(fmt.Sprintf("%v[%d]", test.value, test.index), func(t *testing.T) {
+
+			if test.value[test.index] != 1 {
+				t.Errorf("NewValue failed to set the correct value.")
+			}
+
+		})
+	}
+}
+
+func TestValue_Get(t *testing.T) {
```

Comment:
Please improve the name of this test. There is no function `Get` and even if, a test should be named by the property it is testing, like `TestValue_StringProducesDecimalPrint` below.